table/block_based/block_based_table_reader.h

```diff
@@ -440,6 +440,9 @@ class BlockBasedTable : public TableReader {
                                   char* buffer, size_t* size);
   static void GenerateCachePrefix(Cache* cc, FSWritableFile* file, char* buffer,
                                   size_t* size);
+  static void GeneratePersistentCachePrefix(PersistentCache* cc,
```

Comment:
The function name `GeneratePersistentCachePrefix` can be `GenerateCachePrefix`. We know it's for persistent cache by looking at its first argument. C++ function overloading will take care of it. The implementation of this function does not have anything specific to persistent cache.
One step forward is to use a template function for all the `GeneratePersistentCachePrefix` functions.

Reply:
Good idea, updated, thanks.